lesson05/assignment/index.html

```diff
@@ -9,6 +9,50 @@
     </head>
     <body>
         <main class="main">
+            <div class="container">
+
+               <!-- Adds header element --> 
+              <header>
+                <div class="header">
+                    <h2>Header</h2>
+                </div>
+            </header>
+
+            <!-- Adds nav element with list--> 
+                <nav class="nav">
+                    <ul class="nav__list">
```

Comment:
There is a default 40px left padding on your ul with class nav__list, you might consider setting the padding to overwrite this behavior, as it is causing your links to be shifted off-center to the right.